app/containers/Forms/VerificationStatement.tsx

```diff
@@ -0,0 +1,178 @@
+import { FilePicker } from "@button-inc/bcgov-theme";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import LoadingSpinner from "components/LoadingSpinner";
+import { dateTimeFormat } from "functions/formatDates";
+import createAttachmentMutation from "mutations/application/createAttachmentMutation";
+import deleteAttachmentMutation from "mutations/application/deleteAttachment";
+import React, { useState } from "react";
+import { Col, Row } from "react-bootstrap";
+import { createFragmentContainer, graphql, RelayProp } from "react-relay";
+import { VerificationStatement_application } from "__generated__/VerificationStatement_application.graphql";
+
+function formatBytes(bytes: number, decimals = 2) {
+  if (bytes <= 0) return "0 Bytes";
+
+  const k = 1024;
+  const dm = decimals < 0 ? 0 : decimals;
+  const sizes = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
+
+  const i = Math.floor(Math.log(bytes) / Math.log(k));
+
+  return parseFloat((bytes / k ** i).toFixed(dm)) + " " + sizes[i];
+}
+
+interface Props {
+  application: VerificationStatement_application;
+  relay: RelayProp;
+  onError: () => void;
+}
+export const VerificationStatementComponent: React.FunctionComponent<Props> = ({
+  application,
+  relay,
+  onError,
+}) => {
+  const [isUploading, setIsUploading] = useState(false);
+  const saveAttachment = async (e) => {
+    setIsUploading(true);
+    const { environment } = relay;
+    const file = e.target.files[0];
+    const variables = {
+      connections: [application.attachmentsByApplicationId.__id],
+      input: {
+        attachment: {
+          file,
+          fileName: file.name,
+          fileSize: formatBytes(file.size),
+          fileType: file.type,
+          applicationId: application.rowId,
+          versionNumber: application.latestDraftRevision.versionNumber,
+        },
+      },
+    };
+
+    await createAttachmentMutation(environment, variables)
+      .catch(() => {
+        setIsUploading(false);
+      })
+      .then(() => setIsUploading(false));
+  };
```

Comment:
**To be done in a separate PR:**
We should put in some protections on file size / type. Let's reach out to CAS & determine if they know what file types they expect these verification statements to be formatted as.